contracts/settling_game/library/library_buildings.cairo

```diff
@@ -0,0 +1,224 @@
+# BUILDINGS LIBRARY
+#   functions for
+#
+#
+# MIT License
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.math import unsigned_div_rem, assert_not_zero, assert_le, assert_nn
+from starkware.starknet.common.syscalls import get_block_timestamp
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.registers import get_label_location
+from contracts.settling_game.utils.game_structs import (
+    RealmBuildings,
+    RealmBuildingsSize,
+    BuildingsTime,
+    BuildingsDecaySlope,
+)
+from contracts.settling_game.utils.constants import DAY, BASE_SQM
+from lib.cairo_math_64x61.contracts.Math64x61 import Math64x61_ln
+
+namespace BUILDINGS:
```

Comment:
BUILDINGS -> Buildings
(open for discussion)

Reply:
I'm for of capitalizing these. It makes it easy to read the module files and see what is a lib import

Reply:
It's just that normally you do this for consts, while this is just called as `BUILDINGS.foo()`.